components/cosmetic_filters/browser/cosmetic_filters_resources.cc

```diff
@@ -0,0 +1,123 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 3.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/cosmetic_filters/browser/cosmetic_filters_resources.h"
+
+#include <utility>
+
+#include "base/json/json_reader.h"
+#include "base/optional.h"
+#include "base/values.h"
+#include "brave/components/brave_shields/browser/ad_block_service.h"
+#include "brave/components/brave_shields/browser/brave_shields_util.h"
+#include "components/content_settings/core/browser/host_content_settings_map.h"
+
+namespace cosmetic_filters {
+
+namespace {
+
+base::Optional<base::Value> HiddenClassIdSelectorsOnTaskRunner(
```

Comment:
basically you could post `brave_shields::AdBlockService::HiddenClassIdSelectors` without writing these free functions